hw/ip_templates/alert_handler/data/alert_handler.hjson.tpl

```diff
@@ -234,6 +234,67 @@ chars = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H']
     },
   ]
 
+  features: [
+    { name: "ALERT_HANDLER.ALERT.OBSERVE",
+      desc: '''
+                All incoming alerts can be observed by the alert handler.
+            '''
+    }
+    { name: "ALERT_HANDLER.ALERT.INTERRUPT",
+      desc: '''
+                All alert classes can raise an interrupt when an alert that is
+                allocated to them is raised.
+            '''
+    }
+    { name: "ALERT_HANDLER.ALERT.ESCALATE",
+      desc: '''
+               All incoming alerts can trigger the alert escalation mechanisms
+               of the classes they are allocated to
+            '''
+    }
+    {
+      name: "ALERT_HANDLER.PING_TIMER",
+      descr: '''
+                The alert handler periodicly pings all alert receivers and will
+                raise an alert if the ping isn't responded to in good time
+             '''
+    }
+    { name: "ALERT_HANDLER.ESCALATION.COUNT",
+      descr:
+            '''
+                An escalation can be triggered when the number of alerts seen in
+                a particular class exceeds a software programmable threshold
+            '''
+    }
+    { name: "ALERT_HANDLER.ESCALATION.TIMEOUT",
+      descr:
+            '''
+                An escalation can be triggered when an interrupt from a class
+                isn't acknowledged after a software programmable timeout
+            '''
+    }
+    { name: "ALERT_HANDLER.ESCALATION.PHASES",
+      descr:
+            '''
+               Each alert class can trigger the same 4 escalation phases. A
+               programmable timer specifies the delay between each of the
+               escalation phases. The actions taken on each of the escalation
+               signals is specific to the top-level integration.
+            '''
+    }
+    {
+      name: "ALERT_HANDLER.CRASH_DUMP",
+      descr:
+            '''
+               A crashdump with the state of CSRs and alert handler state bits
+               is made available. When a programmable latching trigger condition
+               is met the crashdump is held constant as its value on that
```

Comment:
Nit: Would it be easier to read as "held constant a**t** its value" ? At a first scan, I thought this said that the crashdump is held constant because its value on that... (and then everything fell apart!)